services/src/messaging/src/main/java/org/openkilda/messaging/command/switches/DefaultRulesAction.java

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.command.switches;
+
+/**
+ * Describes what to do about the switch default rules.
+ */
+public enum DefaultRulesAction {
```

Comment:
Should this enum be embedded with the primary class that uses it?

Reply:
An inner / nested class perfectly suits the case when it's initialized (instantiated) within the parent class only. Otherwise it looks confusing why the 3rd class works with the inner and doesn't involve the parent.
In our case DefaultRulesAction is used by REST resources too.